R/raise_issue.R

```diff
@@ -24,10 +24,7 @@
 #'
 #' @export
 
-raise_issue <- function(
```

Comment:
I think this function can be deleted if we are not automatically logging issues on github, then just revert to using the default error handler on assertr. Alternatively (and preferable) this function can be adapted to include logging using future.logger.